src/subscribables/observable.js

```diff
@@ -63,8 +63,10 @@ ko.hasPrototype = function(instance, prototype) {
 };
 
 ko.isObservable = function (instance) {
-    return ko.hasPrototype(instance, ko.observable);
-}
+    //test if not constructor function;
+    return  ko.hasPrototype(instance, ko.observable) && (instance['fn'] === undefined);
```

Comment:
For older browsers, it's better to use `void 0` instead of `undefined`, or to use the test `typeof instance['fn'] === 'undefined'`, just in case `undefined` is redefined (as can happen prior to ES5, rare though it would be).